moped-editor/src/views/staff/components/NonLoginUserActivationButtons.js

```diff
@@ -60,7 +60,7 @@ const NonLoginUserActivationButtons = ({
   };
 
   const handleUpdateNonLoginUser = async () => {
-    const { password, __typename, ...restOfFormValues } =
+    const { password, __typename, user_id, ...restOfFormValues } =
```

Comment:
removing it from the payload!

Reply:
This is making a lot more sense out of one of the test failures that Rose saw last release. I thought it was stale data only but it was also the mutation not working. So glad this is patched!

Reply:
nice!